src/main/java/nl/enjarai/omnihopper/items/ModItems.java

```diff
@@ -21,13 +22,15 @@ public class ModItems {
     public static final List<Item> ALL = new ArrayList<>();
     public static final List<BlockItem> HOPPERS = ModBlocks.ALL.stream().map(ModItems::registerBlockItem).toList();
 
-    public static void register() {}
+    public static void register() {
+    }
 
     private static BlockItem registerBlockItem(Block block) {
         Identifier id = Registries.BLOCK.getId(block);
-        var item = Registry.register(Registries.ITEM, id, new BlockItem(block, new Item.Settings()) {
+        var item = Registry.register(Registries.ITEM, id, new BlockItem(block, new Item.Settings().registryKey(keyOf(id)).useBlockPrefixedTranslationKey()) {
+
             @Override
-            public void appendTooltip(ItemStack stack, Item.TooltipContext context, List<Text> tooltip, TooltipType type) {
+            public void appendTooltip(ItemStack stack, TooltipContext context, List<Text> tooltip, net.minecraft.item.tooltip.TooltipType type) {
```

Comment:
Why fully qualify the type?

Reply:
Not manually once again, there were errors in the signature, so I commented it out and let idea Regen. Will fix.